Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.13
+
+WORKDIR .
+
+COPY /app/main.py .
```

Comment:
copy folder, rather than a file. you will have to add more and more after you add more files, with folder it is not a problem